ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
I'd do two approval tests to see that a date field with the `{0:g}` display format string outputs with teh correct `data-val-format` attribute - one each for two different UI Cultures so we know it uses the culture from the server.

Make sense?

If you need help writing the tests tests me know.

By the way - I haven't forgotten your other PR, I'm trying to finally schedule some time with @MattDavies to figure it out and get it merged and provide more direction for the rest of the extensibility of this library.